pkg/services/alerting/notifiers/line.go

```diff
@@ -70,27 +77,67 @@ func (this *LineNotifier) createAlert(evalContext *alerting.EvalContext) error {
 		return err
 	}
 
-	form := url.Values{}
-	body := fmt.Sprintf("%s - %s\n%s", evalContext.Rule.Name, ruleUrl, evalContext.Rule.Message)
-	form.Add("message", body)
+	message := fmt.Sprintf("%s - %s\n%s", evalContext.Rule.Name, ruleUrl, evalContext.Rule.Message)
+	var body string
+	header := map[string]string{
+		"Content-Type":  "application/x-www-form-urlencoded;charset=UTF-8",
+		"Authorization": fmt.Sprintf("Bearer %s", this.Token),
+	}
 
+	// send message only if image is not included
+	if !this.UploadImage {
```

Comment:
we don't need this condition since we're setting default body 

```
	// set default body
	form := url.Values{}
	form.Add("message", message)
	body = form.Encode()
```